File: src/main/Java/boj_step10/boj9063.java
import java.io.*;
import java.util.Arrays;
import java.util.StringTokenizer;

public class boj9063 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine());
        int[] xList = new int[n];
        int[] yList = new int[n];
        for (int i = 0; i < n; i++) {
            StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
            xList[i] = Integer.parseInt(tokenizer.nextToken());
            yList[i] = Integer.parseInt(tokenizer.nextToken());
        }
        int width = Arrays.stream(xList).max().getAsInt() - Arrays.stream(xList).min().getAsInt();
        int height = Arrays.stream(yList).max().getAsInt() - Arrays.stream(yList).min().getAsInt();
        writer.append(String.valueOf(width * height)).close();
    }
}

File: src/main/Kotlin/boj_step10/boj9063.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import kotlin.math.max
import kotlin.math.min

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine().toInt()
    var (minx, maxx, miny, maxy) = "0 0 0 0".split(" ").map { it.toInt() }
    repeat(n) {
        val (x, y) = r.readLine().split(" ").map { it.toInt() }
        if(it == 0) {
            minx = x
            maxx = x
            miny = y
            maxy = y
        }
        else {
            minx = min(minx, x)
            maxx = max(maxx, x)
            miny = min(miny, y)
            maxy = max(maxy, y)
        }
    }
    w.write(((maxx - minx) * (maxy - miny)).toString())
    w.close()
 }